KvNGCzA/Hello-Books-BE
# 166214090 Add author endpoint

#### What does this PR do?
It adds the author

#### Description of Task to be completed?
- Create endpoint to add authors
- Create an author controller

#### How should this be manually tested?
- Use **git pull** to pull branch to local branch
- Use **npm install** to download all necessary packages
- Update default config file to use environment variables of the project
- Ensure your database is created or you can use **sequelize db:create** if database does not exist
- Ensure that there is **no id** defined in any models
- Use **npm run db:roll** to create table in database
- Use **npm run test**  to test
- Go your POSTMAN and type **http://localhost:${PORT}/api/v1/admin/author**

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#166214090](https://www.pivotaltracker.com/story/show/166377842)

#### Screenshots (if appropriate)
![authors test](https://user-images.githubusercontent.com/7049701/58757681-3d86fc80-8508-11e9-8e50-c7b874551048.PNG)
Sample of successful author's test

#### Questions:
NA
